lib/pretty_please/prettifier.rb

```diff
@@ -40,8 +40,8 @@ def inspect(object)
 
 		@stack.push(object)
 
-		if object.respond_to?(:pretty_please_inspect)
-			object.pretty_please_inspect(self)
+		if object.respond_to?(:pretty_please)
+			object.pretty_please(self)
```

Comment:
Any thoughts on this interface? Passing `self` makes sense, but what about the method name? `pretty_please` seems like an unlikely conflict.

Reply:
I think that's fair 👍🏼